Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.service.opg.digital/opguk/php-fpm
+FROM registry.service.opg.digital/opguk/php-fpm:0.1.207
```

Comment:
I think this depends on whether its worse to forget to update or to have some instability.